parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -216,9 +216,12 @@ Expand-Archive scripts.zip -DestinationPath "C:\\AzureData\\"
 . c:\AzureData\windows\windowscsehelper.ps1
 # util functions only can be used after this line, for example, Write-Log
 
+$global:OperationId = New-Guid
+$global:HostName = hostname

```

Comment:
I think that we  do not need to define a name and use `$env:COMPUTERNAME` in the code directly.

Reply:
Thanks for the suggestion. Updated it.